values.yaml

```diff
@@ -172,9 +172,9 @@ graphdb:
     # For EE see http://graphdb.ontotext.com/documentation/enterprise/requirements.html
     resources:
       limits:
-        memory: 1Gi
+        memory: 2Gi
```

Comment:
increase it for the workers as well, below :)

Reply:
done